CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Add options for fulfillment create
```

Comment:
Would be useful to note we've added the ability to pass a list of `HTTPoison` options as `options` to `Rest.post` and `Rest.put`.